recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
     name: basemap
-    version: 1.0.8.dev
+    version: 1.0.8.dev0
```

Comment:
If I got PEP 0440 correctly this should not be needed, but conda still installs basemap 1.0.7 in some situations. Not sure if that is a bug in the version solver or something else.

Conda failed to find `basemap 1.0.8.dev` in Windows+Py3k and now on `Travis-CI`.